stob/ad.c

```diff
@@ -109,6 +109,15 @@ M0_TL_DESCR_DEFINE(ad_domains, "ad stob domains", M0_INTERNAL,
 		   M0_AD_DOMAINS_MAGIC, M0_AD_DOMAINS_HEAD_MAGIC);
 M0_TL_DEFINE(ad_domains, M0_INTERNAL, struct ad_domain_map);
 
+M0_INTERNAL uint16_t m0_stob_get_hash(const struct m0_fid *fid)
```

Comment:
I am thinking, why "uint16_t" is used here.
The original thought is that a hash table would have limited entries. Number 65566 (0xFFFF) is big enough.
But I think we can have uint32_t here. The upper limit will be decided by the "EMAP_HT_SIZE", but the size of the return value. 
Otherwise in future, if we happen to increase the hash table size, it would involve more code changes.